src/inline-loader.ts

```diff
@@ -73,31 +74,31 @@ module.exports = async function loader(
       source: string;
       styleId: string;
     };
-    this.resourcePath = source;
+
     const styles = await cache.get(source);
 
-    let css = styles?.get(styleId)?.value;
+    let style = styles?.get(styleId);
```

Comment:
this is the branch that the virtual css files hit. e.g. `inline-loader?styleId=whatever` the ID is used to pull the style from a memory or fs cache. It's a way more complicated base64-loader